src/task.coffee

```diff
@@ -82,8 +82,8 @@ _run = async (ran, name, args) ->
       else
         yield f args...
 
-    finish = Date.now()
-    console.log "Task '#{name}' completed in #{finish - start}ms."
+    # finish = Date.now()
+    # console.log "Task '#{name}' completed in #{finish - start}ms."
```

Comment:
No objection at all, but wasn't sure if @dyoder was done debugging w/e he was debugging there, so I didn't want to tear down [Chesterton's Fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence).